tools/cldr-apps/src/main/java/org/unicode/cldr/web/DBUtils.java

```diff
@@ -139,23 +131,27 @@ public Appendable statsShort(Appendable output) throws IOException {
         return output.append("" + db_number_open).append("/" + db_max_open);
     }
 
+    public static boolean DO_CLOSE = true;
+
```

Comment:
final?

Reply:
should have been deleted.  This was from incorrectly backing out an experiment, I'll refactor this file.